src/sentinel.c

```diff
@@ -241,10 +241,10 @@ typedef struct sentinelValkeyInstance {
     uint64_t failover_epoch; /* Epoch of the currently started failover. */
     int failover_state;      /* See SENTINEL_FAILOVER_STATE_* defines. */
     mstime_t failover_state_change_time;
-    mstime_t failover_start_time;                   /* Last failover attempt start time. */
-    mstime_t failover_timeout;                      /* Max time to refresh failover state. */
-    mstime_t failover_delay_logged;                 /* For what failover_start_time value we
-                                                       logged the failover delay. */
+    mstime_t failover_start_time;                    /* Last failover attempt start time. */
+    mstime_t failover_timeout;                       /* Max time to refresh failover state. */
+    mstime_t failover_delay_logged;                  /* For what failover_start_time value we
+logged the failover delay. */
```

Comment:
Clang-format accepts this? The comment continues on the next line without indentation looks very wrong to me.

Reply:
yes, Clang-format accepts it. The result is due to clang-format, not me.

Reply:
... but it also accepts this?

```suggestion
    mstime_t failover_delay_logged;                  /* For what failover_start_time value we
                                                      * logged the failover delay. */
```

Reply:
it looks like clang-format accepts both. But for your convivence,  I update the format to much more readable.